device/nokia/x86_64-nokia_ixr7250e_36x400g-r0/Nokia-IXR7250E-36x400G/1/jr2cp-nokia-18x400g-config.bcm

```diff
@@ -1265,7 +1265,7 @@ phy_tx_polarity_flip_phy143.BCM8885X=0
 polled_irq_delay.BCM8885X=5
 polled_irq_mode.BCM8885X=0
 port_fec_fabric.BCM8885X=7
-bcm_stat_interval.BCM8885X=1000000
+bcm_stat_interval.BCM8885X=100000
```

Comment:
Hi Vineet this was changed to 1 sec earlier following Broadcom suggestion to reduce the CPU usage. Adding the brcm case CS00011868570  for reference.

Reply:
Do you remember what was the original value? With these settings, i do see 100% CPU util but that is across all cores. per core util is between 30-70%. If there is no functional impact then i believe it is ok to do this change.

Reply:
@judyjoseph , @vmittal-msft , @abdosi ,
Can we converge on the new timer value?  Is this ok to merge?
Included Abhishek in case he has concerns for the other none BRCM based HW SKU where this may potentially causes problem...

Reply:
@gechiang I am working with BRCM to find better solution for this issue.